roles/tas_single_node/templates/manifests/rekor/backfill_redis.sh.j2

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
```

Comment:
While I provided some comments throughout this file, I think a better (although much laborious) route to take would be:
* Create a systemd timer unit
* Create a k8s manifest and use our standard approach to create a systemd service unit for it to run
* Make the timer unit run the job on schedule

Reply:
Im not familiar with systemd timer unit, so i will take a look :) thanks for the feedback